### PR TITLE
vapoursynth 32

### PR DESCRIPTION
**Pull request by ilovezfs**
- patch no longer needed
- update Cython resource
- avoid exceeding 80 characters
- sort the deps

### CODE REVIEW

**Comment by DomT4** [comment]
Patch no longer applies cleanly, may have been merged.

**Comment by ilovezfs** [comment]
@DomT4 PR refreshed.

File: Formula/vapoursynth.rb [hunk 1]
```diff
@@ -11,36 +11,39 @@ class Vapoursynth < Formula
...
-  resource "cython" do
-    url "https://pypi.python.org/packages/source/C/Cython/Cython-0.21.2.tar.gz"
-    sha256 "b01af23102143515e6138a4d5e185c2cfa588e0df61c0827de4257bac3393679"
+  resource "Cython" do
+    url "https://files.pythonhosted.org/packages/b1/51/bd5ef7dff3ae02a2c6047aa18d3d06df2fb8a40b00e938e7ea2f75544cac/Cython-0.24.tar.gz"
+    sha256 "6de44d8c482128efc12334641347a9c3e5098d807dd3c69e867fa8f84ec2a3f1"
   end
 
   def install
+    py3_cfg_prefix = Utils.popen_read("python3-config", "--prefix").chomp
```
Comment (DomT4) [inline Formula/vapoursynth.rb]: These two lines theoretically shouldn't actually be necessary any more. I symlinked both the `python` and `python3` pkg-config files to a sane location a fair while back now.

**Comment by ilovezfs** [comment]
@DomT4 Fixed.

**Comment by DomT4** [comment]
Thanks! LGTM.

**Pull request closed without merging**